```xml
<CALLOUT>
    <CALLOUTMESSAGE>Нарушение общественного порядка</CALLOUTMESSAGE>
    <TASK>
        <TEXT>~o~Бездомный~s~ устроил беспорядок. Поговорите с </TEXT>
        <TEXT>~g~охранником,~s~ именно он вызвал помощь.</TEXT>
        <TEXT>~g~прохожей,~s~ именно она вызвала помощь.</TEXT>
        <TEXT>тем, кто пожаловался.</TEXT>
    </TASK>
    <SCENARIO>
        <SEARCHINGTRASH>
            <COOPERATIVE>
                <TEXT>~y~Вы:~s~ Здравствуйте. Всё в порядке? Вы тут роетесь в мусоре.</TEXT>
                <TEXT>~o~Бездомный:~s~ Да... просто бутылки ищу, и всё. Никому не мешаю.</TEXT>
                <TEXT>~y~Вы:~s~ Я понимаю. Но это частная территория. Не могли бы вы пройти дальше?</TEXT>
                <TEXT>~o~Бездомный:~s~ Ох... извините, не знал. Я никого не хотел беспокоить.</TEXT>
                <TEXT>~y~Вы:~s~ Всё в порядке. Вам что-нибудь нужно? Кофе? Помощь соцслужб?</TEXT>
                <TEXT>~o~Бездомный:~s~ Это очень любезно, но нет... Я сейчас пойду.</TEXT>
                <TEXT>~y~Вы:~s~ Хорошо. Берегите себя.</TEXT>
                <TEXT>~o~Бездомный:~s~ Спасибо. Вам тоже.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Вы:~s~ Прошу прощения. Вам нельзя здесь находиться, это частная территория.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я тут уже несколько дней. Никому не мешаю.</TEXT>
                <TEXT>~y~Вы:~s~ Поступали жалобы. Мне нужно, чтобы вы прошли дальше.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я никуда не уйду. Это моё единственное место.</TEXT>
                <TEXT>~y~Вы:~s~ Если прибудет охрана, для вас это может усложниться.</TEXT>
                <TEXT>~o~Бездомный:~s~ Пусть приходят. Я остаюсь здесь.</TEXT>
                <TEXT>~y~Вы:~s~ Я даю вам шанс решить это мирно.</TEXT>
                <TEXT>~o~Бездомный:~s~ А я говорю «нет». Оставьте меня в покое.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Вы:~s~ Эй, можете на секунду остановиться? Люди жалуются, что вы тут роетесь.</TEXT>
                <TEXT>~o~Бездомный:~s~ Жалуются? На что? Я просто пытаюсь выжить. Отстаньте от меня.</TEXT>
                <TEXT>~y~Вы:~s~ Я хочу решить это мирно. Пожалуйста, пройдите дальше.</TEXT>
                <TEXT>~o~Бездомный:~s~ Нет. Я никуда не уйду.</TEXT>
                <TEXT>~y~Вы:~s~ Последнее предупреждение. Не усложняйте.</TEXT>
                <TEXT>~o~Бездомный:~s~ Усложняю? Вы мне не страшны.</TEXT>
                <TEXT>~y~Вы:~s~ Назад. Немедленно.</TEXT>
                <TEXT>~o~Бездомный:~s~ Ну всё! Я тебя сейчас!</TEXT>
                <TEXT>~y~Вы:~s~ Прекратите! Немедленно!</TEXT>
            </FIGHT>
        </SEARCHINGTRASH>
        <BEGFORMONEY>
            <INTRO>
                <TEXT>~y~Вы:~s~ Здравствуйте, я по вызову. Вы сообщали, что вас кто-то донимает?</TEXT>
                <TEXT>~g~Горожанин:~s~ Да, офицер. Вон тот бездомный постоянно просит у меня деньги.</TEXT>
                <TEXT>~y~Вы:~s~ Расскажите, что именно произошло?</TEXT>
                <TEXT>~g~Горожанин:~s~ Я просто шёл мимо, занимался своими делами, и он вдруг остановил меня.</TEXT>
                <TEXT>~g~Горожанин:~s~ Он продолжал настаивать, говорил, что «это ему нужнее, чем мне». Не принимал «нет» за ответ.</TEXT>
                <TEXT>~y~Вы:~s~ Он вам угрожал или прикасался к вам?</TEXT>
                <TEXT>~g~Горожанин:~s~ Нет, не совсем... но он был слишком навязчив. Мне было не по себе.</TEXT>
                <TEXT>~g~Горожанин:~s~ Я понимаю, времена тяжёлые, но меня не должны донимать просто за то, что я иду по улице.</TEXT>
                <TEXT>~y~Вы:~s~ Хорошо, я поговорю с ним. Спасибо, что сообщили.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~Вы:~s~ Эй, мне нужно с вами поговорить. Это вы просили у той женщины деньги?</TEXT>
                <TEXT>~o~Бездомный:~s~ Да... Я только раз спросил. Я не хотел её пугать.</TEXT>
                <TEXT>~y~Вы:~s~ Ей было некомфортно. Нельзя так обращаться к людям.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я понял, понял... Я не хотел создавать проблем.</TEXT>
                <TEXT>~y~Вы:~s~ Хорошо. Мне нужно, чтобы вы ушли отсюда и перестали просить деньги у незнакомцев.</TEXT>
                <TEXT>~o~Бездомный:~s~ Ладно... Я пройду дальше. Просто пытаюсь выжить, понимаете?</TEXT>
                <TEXT>~y~Вы:~s~ Я понимаю. Но здесь этого делать нельзя.</TEXT>
                <TEXT>~o~Бездомный:~s~ Хорошо... Я ухожу. Спасибо, что не устроили скандал.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Вы:~s~ Гражданин, мне нужно с вами поговорить. Вы просили у людей деньги.</TEXT>
                <TEXT>~o~Бездомный:~s~ Ну и что? Я никого не тронул. Я просто спросил.</TEXT>
                <TEXT>~y~Вы:~s~ Вы заставили женщину чувствовать себя неловко. Вам нужно пройти дальше.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я никуда не уйду. Люди ходят тут весь день, я могу спрашивать у кого хочу.</TEXT>
                <TEXT>~y~Вы:~s~ Так не пойдёт. Нельзя приставать к прохожим.</TEXT>
                <TEXT>~o~Бездомный:~s~ Приставать? Я едва слово сказал. Она преувеличивает.</TEXT>
                <TEXT>~y~Вы:~s~ Последнее предупреждение: прекратите попрошайничать и покиньте это место.</TEXT>
                <TEXT>~o~Бездомный:~s~ Нет. Я остаюсь. Мне больше некуда идти.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Вы:~s~ Мне нужно, чтобы вы перестали донимать людей, выпрашивая деньги. Здесь этого делать нельзя.</TEXT>
                <TEXT>~o~Бездомный:~s~ Донавил? Я просто задал вопрос. Люди стали чертовски чувствительными.</TEXT>
                <TEXT>~y~Вы:~s~ Она вызвала нас, потому что почувствовала угрозу. Вам нужно уйти, сейчас же.</TEXT>
                <TEXT>~o~Бездомный:~s~ Уйти? А может, это вы отстанете от меня?</TEXT>
                <TEXT>~y~Вы:~s~ Не усугубляйте. Просто уйдите.</TEXT>
                <TEXT>~o~Бездомный:~s~ Уйти? Я сыт по горло тем, что меня гонят!</TEXT>
                <TEXT>~y~Вы:~s~ Эй — успокойтесь.</TEXT>
                <TEXT>~o~Бездомный:~s~ Нет! Я больше не слушаю!</TEXT>
                <TEXT>~o~Бездомный:~s~ Давай! Посмотрим, на что ты способен!</TEXT>
                <TEXT>~y~Вы:~s~ Стоп! Не делайте этого!</TEXT>
            </FIGHT>
        </BEGFORMONEY>
        <SLEEPONSTREET>
            <INTRO>
                <TEXT>~y~Вы:~s~ Здравствуйте, я по вашему вызову. Вы говорили, что кто-то лежит на земле?</TEXT>
                <TEXT>~g~Горожанин:~s~ Да, офицер. Прямо там. Он спит на улице уже как минимум час.</TEXT>
                <TEXT>~y~Вы:~s~ Вы видели, чтобы он двигался? Может, есть признаки, что он ранен?</TEXT>
                <TEXT>~g~Горожанин:~s~ Не особо... Я не был уверен, что с ним всё в порядке. Поэтому и позвонил. Выглядело это ненормально.</TEXT>
                <TEXT>~y~Вы:~s~ Я понимаю. Спасибо, что сообщили. Мы его проверим и убедимся, что он в безопасности.</TEXT>
                <TEXT>~g~Горожанин:~s~ Я просто не хотел, чтобы его задавили или что-то вроде того. Машины проезжают тут совсем рядом.</TEXT>
                <TEXT>~y~Вы:~s~ Вы поступили правильно. Я займусь этим и выясню, нужна ли ему медицинская помощь или просто нужно пройти дальше.</TEXT>
                <TEXT>~g~Горожанин:~s~ Ладно... спасибо, офицер.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~Вы:~s~ Эй, можете проснуться? Мне нужно вас проверить.</TEXT>
                <TEXT>~o~Бездомный:~s~ М-м?.. Что?.. А, офицер... Я, наверное, задремал.</TEXT>
                <TEXT>~y~Вы:~s~ Кто-то позвонил, потому что беспокоился о вашем здоровье. Вы лежали прямо у проезжей части.</TEXT>
                <TEXT>~o~Бездомный:~s~ Да... Я устал. Не думал, что это кого-то побеспокоит.</TEXT>
                <TEXT>~y~Вы:~s~ Я понимаю, но здесь опасно. Машины проезжают близко, вас могут сбить.</TEXT>
                <TEXT>~o~Бездомный:~s~ Ладно... ладно... Я встаю. Не хотел создавать проблем.</TEXT>
                <TEXT>~y~Вы:~s~ Хорошо. Постарайтесь в следующий раз найти место поспокойнее.</TEXT>
                <TEXT>~o~Бездомный:~s~ Обязательно. Спасибо, что проверили.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Вы:~s~ Гражданин, проснитесь. Мне нужно убедиться, что с вами всё в порядке.</TEXT>
                <TEXT>~o~Бездомный:~s~ Чего?.. Я в порядке... отстаньте... просто сплю.</TEXT>
                <TEXT>~y~Вы:~s~ Кто-то позвонил, потому что подумал, что вы могли пострадать. Вы лежите прямо у дороги.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я не пострадал. Я сам выбрал это место. Тут тихо. Я остаюсь.</TEXT>
                <TEXT>~y~Вы:~s~ Здесь небезопасно. Вы не можете здесь спать, машины проезжают опасно близко.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я спал и в худших местах. Я не сдвинусь.</TEXT>
                <TEXT>~y~Вы:~s~ Я даю вам шанс переселиться самостоятельно.</TEXT>
                <TEXT>~o~Бездомный:~s~ А я говорю вам «нет». Просто дайте мне поспать.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Вы:~s~ Гражданин, проснитесь. Мне нужно вас проверить и убедиться, что вы в безопасности.</TEXT>
                <TEXT>~o~Бездомный:~s~ Какого чёрта?.. Почему вы меня донимаете?</TEXT>
                <TEXT>~y~Вы:~s~ Кто-то беспокоился о вашем состоянии. Вы лежали прямо у потока машин.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я спал! Это же не преступление!</TEXT>
                <TEXT>~y~Вы:~s~ Это для вашей же безопасности. Мне нужно, чтобы вы встали и перешли в более безопасное место.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я никуда не уйду! Отстаньте от меня!</TEXT>
                <TEXT>~y~Вы:~s~ Успокойтесь. Не стоит делать из этого того, чем это не является.</TEXT>
                <TEXT>~o~Бездомный:~s~ Нет! Я сыт по горло тем, что меня гонят!</TEXT>
                <TEXT>~o~Бездомный:~s~ Отойдите от меня!</TEXT>
                <TEXT>~y~Вы:~s~ Не делайте этого! Прекратите немедленно!</TEXT>
            </FIGHT>
        </SLEEPONSTREET>
        <TRESPASSING>
            <INTRO>
                <TEXT>~g~Охрана:~s~ Офицер, спасибо, что быстро приехали.</TEXT>
                <TEXT>~g~Охрана:~s~ У нас тут бездомный спит, прямо вон там.</TEXT>
                <TEXT>~g~Охрана:~s~ Он тут уже какое-то время, а это частная территория. Он не может здесь оставаться.</TEXT>
                <TEXT>~g~Охрана:~s~ Я пытался его разбудить и попросить уйти, но он почти не отреагировал.</TEXT>
                <TEXT>~g~Охрана:~s~ Честно говоря, я не уверен в его состоянии. Может, он в порядке, но я не хотел рисковать.</TEXT>
                <TEXT>~g~Охрана:~s~ Не могли бы вы поговорить с ним и проследить, чтобы он покинул территорию?</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~Вы:~s~ Эй, можете проснуться? Охрана вызвала нас, потому что вы на частной территории.</TEXT>
                <TEXT>~o~Бездомный:~s~ М-м?.. Что? Извините, я не хотел создавать проблем.</TEXT>
                <TEXT>~y~Вы:~s~ У вас нет проблем, но оставаться здесь нельзя. Они также беспокоились о вашем состоянии.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я понимаю... Я просто выбился из сил. Я сейчас уйду.</TEXT>
                <TEXT>~y~Вы:~s~ Хорошо. Если вам нужна помощь или более безопасное место, я могу подсказать, куда обратиться.</TEXT>
                <TEXT>~o~Бездомный:~s~ Благодарю, офицер. Я сейчас выйду.</TEXT>
                <TEXT>~y~Вы:~s~ Хорошо. Берегите себя, ладно?</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Вы:~s~ Гражданин, проснитесь. Вы на частной территории и не можете здесь оставаться.</TEXT>
                <TEXT>~o~Бездомный:~s~ Почему... почему я не могу просто отдохнуть? Я никому не мешаю.</TEXT>
                <TEXT>~y~Вы:~s~ Охрана беспокоилась о вашем самочувствии и о том, что вы нарушаете границы.</TEXT>
                <TEXT>~o~Бездомный:~s~ Это просто место, чтобы поспать. Я ничему не вредю.</TEXT>
                <TEXT>~y~Вы:~s~ Я понимаю, что вы устали, но вам всё равно нужно пройти дальше. Эта территория не для публики.</TEXT>
                <TEXT>~o~Бездомный:~s~ Не понимаю, почему я должен уходить... здесь тихо.</TEXT>
                <TEXT>~y~Вы:~s~ Я прошу вас в последний раз — давайте без осложнений. Вам нужно встать и очистить территорию.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Вы:~s~ Гражданин, проснитесь. Вы нарушаете границы и не можете здесь оставаться.</TEXT>
                <TEXT>~o~Бездомный:~s~ Отвали... я никуда не уйду!</TEXT>
                <TEXT>~y~Вы:~s~ Охрана беспокоилась, что что-то не так. Я пытаюсь вам помочь, но вам нужно уйти.</TEXT>
                <TEXT>~o~Бездомный:~s~ Помочь? Вы, полицейские, никогда не помогаете! Просто отстаньте от меня!</TEXT>
                <TEXT>~y~Вы:~s~ Успокойтесь. Я отдаю вам законное распоряжение встать и покинуть территорию.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я сказал нет! Отойдите от меня!</TEXT>
                <TEXT>~y~Вы:~s~ Не делайте того, о чём пожалеете — прекратите сопротивление!</TEXT>
            </FIGHT>
        </TRESPASSING>
        <FIREPLACE>
            <INTRO>
                <TEXT>~g~Охрана:~s~ Офицер, спасибо, что приехали. У нас снова на территории бездомный.</TEXT>
                <TEXT>~g~Охрана:~s~ На этот раз он развёл маленький костёр, чтобы согреться, прямо у стены здания.</TEXT>
                <TEXT>~y~Вы:~s~ Ясно, это определённо небезопасно. Кто-нибудь ещё рядом или только он?</TEXT>
                <TEXT>~g~Охрана:~s~ Только он, но дым сработал на одном из наших датчиков. И он отказался уходить, когда я его попросил.</TEXT>
                <TEXT>~y~Вы:~s~ Понял. Я поговорю с ним и возьму ситуацию под контроль.</TEXT>
                <TEXT>~g~Охрана:~s~ Пожалуйста. Он не может здесь оставаться, и этот огонь — настоящая угроза.</TEXT>
                <TEXT>~y~Вы:~s~ Я понимаю. Я прослежу, чтобы он потушил огонь и ушёл.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~Вы:~s~ Эй, гражданин. Мне нужно, чтобы вы отошли от огня.</TEXT>
                <TEXT>~o~Бездомный:~s~ Ох... офицер. Я просто пытался согреться.</TEXT>
                <TEXT>~y~Вы:~s~ Я понимаю, но это частная территория, и этот огонь представляет серьёзную опасность.</TEXT>
                <TEXT>~o~Бездомный:~s~ Да... извините. Не хотел никому мешать.</TEXT>
                <TEXT>~y~Вы:~s~ Всё в порядке. Мне просто нужно, чтобы вы потушили огонь и прошли дальше.</TEXT>
                <TEXT>~o~Бездомный:~s~ Ладно, ладно. Я уйду. Я не хочу никаких проблем.</TEXT>
                <TEXT>~y~Вы:~s~ Хорошо. Берегите себя и оставайтесь в безопасности.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Вы:~s~ Гражданин, мне нужно, чтобы вы потушили огонь и отошли от этого места.</TEXT>
                <TEXT>~o~Бездомный:~s~ Почему? Я никому не мешаю. Мне просто нужно было согреться.</TEXT>
                <TEXT>~y~Вы:~s~ Это частная территория, и вы не можете здесь оставаться. К тому же, огонь опасен.</TEXT>
                <TEXT>~o~Бездомный:~s~ Всё под контролем. Я знаю, что делаю.</TEXT>
                <TEXT>~y~Вы:~s~ Это не меняет ситуацию. Вам нужно пройти дальше.</TEXT>
                <TEXT>~o~Бездомный:~s~ Не понимаю, почему я должен уходить. Всего на одну ночь.</TEXT>
                <TEXT>~y~Вы:~s~ Я отдаю вам законное распоряжение. Не заставляйте усложнять.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Вы:~s~ Гражданин, полиция. Отойдите от огня и поговорите со мной.</TEXT>
                <TEXT>~o~Бездомный:~s~ Отвали... я никуда не уйду!</TEXT>
                <TEXT>~y~Вы:~s~ Это частная территория, и этот огонь представляет угрозу. Вам нужно потушить его сейчас же.</TEXT>
                <TEXT>~o~Бездомный:~s~ Я не собираюсь выполнять ваши приказы! Отстаньте от меня!</TEXT>
                <TEXT>~y~Вы:~s~ Успокойтесь. Я пытаюсь решить это мирно.</TEXT>
                <TEXT>~o~Бездомный:~s~ Мирно? Вы здесь для того, чтобы выгнать меня! Забудьте!</TEXT>
                <TEXT>~y~Вы:~s~ Не делайте этого. Руки, где я их вижу — сейчас же!</TEXT>
            </FIGHT>
        </FIREPLACE>
    </SCENARIO>
</CALLOUT>