<CALLOUT>
    <CALLOUTMESSAGE>Rusení porádku bezdomovci</CALLOUTMESSAGE>
    <TASK>
        <TEXT>~o~Bezdomovec~s~ zpusobuje výtrznosti. Promluvte si s ním.</TEXT>
        <TEXT>~g~Ochranka,~s~ zavolala o pomoc.</TEXT>
        <TEXT>~g~Chodec,~s~ zavolal o pomoc.</TEXT>
        <TEXT>je.</TEXT>
    </TASK>
    <SCENARIO>
        <SEARCHINGTRASH>
            <COOPERATIVE>
                <TEXT>~y~Ty:~s~ Ahoj. Vsechno v porádku? Prohrabávás se tady odpadky.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Jo... jen hledám láhve, to je vsechno. Nechci delat potíze.</TEXT>
                <TEXT>~y~Ty:~s~ Chápu. Ale tohle je soukromý pozemek. Mohl bys odejít?</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Oh... promin, to jsem nevedel. Nechci nikoho rozcilovat.</TEXT>
                <TEXT>~y~Ty:~s~ To je v porádku. Potrebujete neco? Kávu? Pomoc s nejakými sluzbami?</TEXT>
                <TEXT>~o~Bezdomovec:~s~ To je od vás milé, ale ne... uz pujdu.</TEXT>
                <TEXT>~y~Ty:~s~ Dobre. Opatrujte se.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Vy taky.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Ty:~s~ Prominte. Tady nemuzete zustat, je to soukromý pozemek.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Jsem tu uz nekolik dní. Nikomu nevadím.</TEXT>
                <TEXT>~y~Ty:~s~ Byly stíznosti. Musíte odejít.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Nikam nepujdu. Tohle je jediné místo, které mám.</TEXT>
                <TEXT>~y~Ty:~s~ Pokud prijede ostraha, mohlo by to pro vás být nepríjemné.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ At si prijedou. Já zustanu tady.</TEXT>
                <TEXT>~y~Ty:~s~ Dávám vám férovou sanci.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ A já ríkám ne. Nechte me být.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Ty:~s~ Hej, muzete na chvíli prestat? Lidé si stezovali, ze tady kopete.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Stezovali? Na co? Jen se snazím prezít. Nechte me na pokoji.</TEXT>
                <TEXT>~y~Ty:~s~ Chci, aby tu byl klid. Prosím, odejdete.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Ne. Nikam nepujdu.</TEXT>
                <TEXT>~y~Ty:~s~ Poslední varování. Neztezuj si to víc, nez je nutné.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Ztezovat? Ty me nevydesís.</TEXT>
                <TEXT>~y~Ty:~s~ Ustup. Hned.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ To stací! Rozbiju ti hlavu!</TEXT>
                <TEXT>~y~Ty:~s~ Stuj! Hned!</TEXT>
            </FIGHT>
        </SEARCHINGTRASH>
        <BEGFORMONEY>
            <INTRO>
                <TEXT>~y~Ty:~s~ Dobrý den, volám kvuli tomu telefonátu. Nahlásil jste, ze vás nekdo obtezuje?</TEXT>
                <TEXT>~g~Civilista:~s~ Ano, pane policisto. Ten bezdomovec tamhle me porád zádá o peníze.</TEXT>
                <TEXT>~y~Ty:~s~ Muzete mi presne popsat, co se stalo?</TEXT>
                <TEXT>~g~Civilista:~s~ Procházela jsem kolem, starala se o své veci, a on me z niceho nic zastavil.</TEXT>
                <TEXT>~g~Civilista:~s~ Neustále naléhal a ríkal, ze „to potrebuje víc nez já“. Nechtel prijmout odmítnutí.</TEXT>
                <TEXT>~y~Ty:~s~ Vyhrozoval vám nebo se vás nejak dotýkal?</TEXT>
                <TEXT>~g~Civilista:~s~ Ne, ne tak úplne... ale byl prílis vlezlý. Cítil jsem se nepríjemne.</TEXT>
                <TEXT>~g~Civilista:~s~ Chápu, ze jsou tezké casy, ale nemel bych být obtezován jen proto, ze jdu po ulici.</TEXT>
                <TEXT>~y~Ty:~s~ Dobre, pujdu si s ním promluvit. Dekuji za informaci.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~Ty:~s~ Hele, musím s tebou mluvit. Ty jsi zádal tu zenu o peníze?</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Jo... Zeptal jsem se jen jednou. Nechtel jsem ji vydesit.</TEXT>
                <TEXT>~y~Ty:~s~ Cítila se nepríjemne. Takhle se k lidem chovat nemuzes.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Chápu, chápu... Nechtel jsem delat potíze.</TEXT>
                <TEXT>~y~Ty:~s~ Dobre. Potrebuju, abys odesel z této oblasti a prestal zádat cizí lidi o peníze.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Dobre... Pujdu pryc. Jen se snazím prezít, vís?</TEXT>
                <TEXT>~y~Ty:~s~ Chápu. Ale tady to delat nemuzes.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Dobre... Jdu pryc. Díky, ze z toho nedelás velkou vedu.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Ty:~s~ Pane, musím s vámi mluvit. zádáte lidi o peníze.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ No a co? Nikomu jsem neublízil. Jen jsem se zeptal.</TEXT>
                <TEXT>~y~Ty:~s~ Znepokojil jste jednu zenu. Musíte odejít.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Nikam nejdu. Lidé tu chodí celý den, muzu se zeptat, koho chci.</TEXT>
                <TEXT>~y~Ty:~s~ Tak to nefunguje. Nemuzete obtezovat kolemjdoucí.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Obtezovat? Sotva jsem neco rekl. Ona prehání.</TEXT>
                <TEXT>~y~Ty:~s~ Toto je vase poslední varování: prestante tu zebrat a opustte tuto oblast.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Ne. Zustanu. Nemám kam jít.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Ty:~s~ Musís prestat obtezovat lidi kvuli penezum. Tady to nemuzes delat.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Obtezovat? Jen jsem se zeptal na jednoduchou otázku. Lidi jsou tak zatracene citliví.</TEXT>
                <TEXT>~y~Ty:~s~ Zavolala nás, protoze se cítila ohrozená. Musís odejít, hned.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Odejít? Proc radsi nenechás na pokoji me?</TEXT>
                <TEXT>~y~Ty:~s~ Neeskalujte to. Proste odejdete.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Odejít? Mám dost toho, jak se mnou zametají!</TEXT>
                <TEXT>~y~Ty:~s~ Hej, uklidnete se.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Ne! Uz vás nebudu poslouchat!</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Tak pojdte! Ukazte, co umíte!</TEXT>
                <TEXT>~y~Ty:~s~ Prestante! Nedelejte to!</TEXT>
            </FIGHT>
        </BEGFORMONEY>
        <SLEEPONSTREET>
            <INTRO>
                <TEXT>~y~Ty:~s~ Dobrý den, odpovídám na vás hovor. ríkal jste, ze nekdo lezí na zemi?</TEXT>
                <TEXT>~g~Civilista:~s~ Ano, pane policisto. Tamhle. Spí na ulici uz nejméne hodinu.</TEXT>
                <TEXT>~y~Ty:~s~ Videl jste, ze by se pohnul? Vypadá to, ze by mohl být zranený?</TEXT>
                <TEXT>~g~Civilista:~s~ Ne, ani ne... Nebyl jsem si jistý, jestli je v porádku. Proto jsem volal. Nevypadalo to normálne.</TEXT>
                <TEXT>~y~Ty:~s~ Rozumím. Dekujeme za informaci. Zkontrolujeme ho a ujistíme se, ze je v bezpecí.</TEXT>
                <TEXT>~g~Civilista:~s~ Jen jsem nechtel, aby ho nekdo prejel nebo tak neco. Auta tady jezdí dost blízko.</TEXT>
                <TEXT>~y~Ty:~s~ Udelal jste správnou vec. Prevezmu to a zjistím, jestli potrebuje lékarskou pomoc, nebo se jen potrebuje presunout.</TEXT>
                <TEXT>~g~Civilista:~s~ Dobre... dekuji, pane policisto.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~Ty:~s~ Hej, muzete se probudit? Musím vás zkontrolovat.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Hm…? Co…? Ach, pane policisto… Musel jsem usnout.</TEXT>
                <TEXT>~y~Ty:~s~ Nekdo volal, protoze mel obavy o vase zdraví. Lezel jste prímo u silnice.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Jo… Byl jsem unavený. Nemyslel jsem si, ze je to neco vázného.</TEXT>
                <TEXT>~y~Ty:~s~ Chápu, ale tady je to nebezpecné. Auta jezdí blízko, mohl byste se zranit.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Dobre... dobre... Vstanu. Nechtel jsem delat potíze.</TEXT>
                <TEXT>~y~Ty:~s~ Dobre. Príste si zkuste najít bezpecnejsí místo.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Dobre. Dekuji, ze jste me zkontroloval.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Ty:~s~ Pane, probudte se. Musím se ujistit, ze jste v porádku.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Co…? Jsem v porádku… nechte me být… jen se snazím spát.</TEXT>
                <TEXT>~y~Ty:~s~ Nekdo volal, protoze si myslel, ze jste se mozná zranil. Jste prímo u silnice.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Nejsem zranený. Vybral jsem si tohle místo. Je tu ticho. Zustanu tady.</TEXT>
                <TEXT>~y~Ty:~s~ Tohle místo není bezpecné. Nemuzete tu spát, auta projízdejí nebezpecne blízko.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Spal jsem na horsích místech. Nehodlám se hnout.</TEXT>
                <TEXT>~y~Ty:~s~ Dávám vám sanci, abyste se prestehoval sám.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ A já vám ríkám ne. Nechte me spát.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Ty:~s~ Pane, probudte se. Musím vás zkontrolovat a ujistit se, ze jste v bezpecí.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Co to sakra…? Proc me obtezujete?</TEXT>
                <TEXT>~y~Ty:~s~ Nekdo mel obavy o vás stav. Lezel jste prímo u silnice.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Spal jsem! To není zlocin!</TEXT>
                <TEXT>~y~Ty:~s~ Je to pro vasi bezpecnost. Potrebuji, abyste vstal a presunul se na bezpecnejsí místo.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Nikam nepujdu! Nechte me na pokoji!</TEXT>
                <TEXT>~y~Ty:~s~ Uklidnete se. Nedelejte z toho neco, co to není.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Ne! Mám dost toho, jak se mnou nekdo manipuluje!</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Jdete ode me pryc!</TEXT>
                <TEXT>~y~Ty:~s~ Nedelejte to! Okamzite ustupte!</TEXT>
            </FIGHT>
        </SLEEPONSTREET>
        <TRESPASSING>
            <INTRO>
                <TEXT>~g~Ochranka:~s~ Pane policisto, dekuji, ze jste prisel tak rychle.</TEXT>
                <TEXT>~g~Ochranka:~s~ Máme tu bezdomovce, který spí prímo tamhle.</TEXT>
                <TEXT>~g~Ochranka:~s~ Je tu uz nejakou dobu a tato oblast je soukromý pozemek. Nemuze tu zustat.</TEXT>
                <TEXT>~g~Ochranka:~s~ Snazil jsem se ho vzbudit a pozádat ho, aby odesel, ale témer nereagoval.</TEXT>
                <TEXT>~g~Ochranka:~s~ Uprímne receno, nejsem si jistý, v jakém je stavu. Mozná je v porádku, ale nechtel jsem to riskovat.</TEXT>
                <TEXT>~g~Ochranka:~s~ Mohl byste s ním promluvit a zajistit, aby opustil tento pozemek?</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~Ty:~s~ Ahoj, muzes se probudit? Ochranka nám volala, protoze jsi na soukromém pozemku.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Coze? Promin, nechtel jsem delat potíze.</TEXT>
                <TEXT>~y~Ty:~s~ Nemás zádné potíze, ale nemuzes tady zustat. Také se obávali o tvuj stav.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Chápu... Byl jsem jen vycerpaný. Pujdu.</TEXT>
                <TEXT>~y~Ty:~s~ Dobre. Pokud potrebujes pomoc nebo bezpecnejsí místo, kde bys mohl být, mohu ti doporucit nekteré sluzby.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Dekuji, pane policisto. Ted uz pujdu.</TEXT>
                <TEXT>~y~Ty:~s~ Dobre. Dávej na sebe pozor, ano?</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Ty:~s~ Pane, probudte se. Jste na soukromém pozemku a nemuzete tu zustat.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Proc... proc si tu nemuzu odpocinout? Nikomu nevadím.</TEXT>
                <TEXT>~y~Ty:~s~ Ostraha se obává o vase zdraví a také o to, ze jste vnikl na cizí pozemek.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Je to jen místo na spaní. Nikomu neublizuji.</TEXT>
                <TEXT>~y~Ty:~s~ Chápu, ze jste unavený, ale presto musíte odejít. Tato oblast není prístupná verejnosti.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Nevidím duvod, proc bych mel odejít... tady je klid.</TEXT>
                <TEXT>~y~Ty:~s~ zádám vás naposledy – ulehcete nám to. Musíte vstát a opustit tuto oblast.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Ty:~s~ Pane, probudte se. Jste na cizím pozemku a nemuzete tu zustat.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Nechte me být... Nikam nejdu!</TEXT>
                <TEXT>~y~Ty:~s~ Ostraha mela obavy, ze se neco stalo. Snazím se vám pomoct, ale musíte odejít.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Pomoc? Vy policajti nikdy nepomáháte! Nechte me na pokoji!</TEXT>
                <TEXT>~y~Ty:~s~ Uklidnete se. Dávám vám zákonný príkaz, abyste vstal a opustil toto místo.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ rekl jsem ne! Nechte me být!</TEXT>
                <TEXT>~y~Ty:~s~ Nedelejte nic, ceho byste litoval – prestante klást odpor!</TEXT>
            </FIGHT>
        </TRESPASSING>
        <FIREPLACE>
            <INTRO>
                <TEXT>~g~Ochranka:~s~ Pane policisto, dekujeme, ze jste prisel. Na pozemku máme zase bezdomovce.</TEXT>
                <TEXT>~g~Ochranka:~s~ Tentokrát dokonce rozdelal malý ohen, aby se zahrál, prímo u budovy.</TEXT>
                <TEXT>~y~Ty:~s~ Dobre, to rozhodne není bezpecné. Je tam jeste nekdo dalsí, nebo je tam jen on?</TEXT>
                <TEXT>~g~Ochranka:~s~ Jen on, ale kour spustil jeden z nasich senzoru. A kdyz jsem ho pozádal, aby odesel, odmítl.</TEXT>
                <TEXT>~y~Ty:~s~ Rozumím. Promluvím s ním a situaci vyresím.</TEXT>
                <TEXT>~g~Ochranka:~s~ Prosím, udelejte to. Nemuze tu zustat a ten ohen je opravdu nebezpecný.</TEXT>
                <TEXT>~y~Ty:~s~ Rozumím. Zajistím, aby ohen uhasil a odesel.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~Ty:~s~ Hej, pane. Potrebuji, abyste se vzdálil od ohne.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Oh... strázníku. Jen jsem se snazil zahrát.</TEXT>
                <TEXT>~y~Ty:~s~ Chápu, ale tohle je soukromý pozemek a ten ohen predstavuje vázné nebezpecí.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Jo... omlouvám se. Nechtel jsem zpusobit zádné potíze.</TEXT>
                <TEXT>~y~Ty:~s~ To je v porádku. Jen potrebuji, abyste ohen uhasil a odesel.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Dobre, dobre. Odejdeme. Nechci zádné problémy.</TEXT>
                <TEXT>~y~Ty:~s~ Dobre. Opatrujte se a budte opatrný.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~Ty:~s~ Pane, potrebuji, abyste uhasil ohen a opustil tuto oblast.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Proc? Nikomu neublizuji. Jen jsem potreboval teplo.</TEXT>
                <TEXT>~y~Ty:~s~ Toto je soukromý pozemek a nemuzete tu zustat. Ohen je také nebezpecný.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Mám to pod kontrolou. Vím, co delám.</TEXT>
                <TEXT>~y~Ty:~s~ To nic nemení na situaci. Musíte odejít.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Nechápu, proc musím odejít. Je to jen jedna noc.</TEXT>
                <TEXT>~y~Ty:~s~ Dávám vám zákonný príkaz. Neztezujte si to víc, nez je nutné.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~Ty:~s~ Pane, policie. Odstupte od ohne a promluvte si se mnou.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Jdete pryc... Nikam nepujdu!</TEXT>
                <TEXT>~y~Ty:~s~ Toto je soukromý pozemek a ten ohen je nebezpecný. Musíte ho okamzite uhasit.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Nebudu poslouchat vase rozkazy! Nechte me na pokoji!</TEXT>
                <TEXT>~y~Ty:~s~ Uklidnete se. Snazím se to vyresit mírumilovne.</TEXT>
                <TEXT>~o~Bezdomovec:~s~ Mírumilovne? Jste tu, abyste me vyhodili! Zapomente na to!</TEXT>
                <TEXT>~y~Ty:~s~ Nedelejte to. Dejte ruce tak, abych na ne videl – hned!</TEXT>
            </FIGHT>
        </FIREPLACE>
    </SCENARIO>
</CALLOUT>
