<CALLOUT>
    <CALLOUTMESSAGE>流浪汉扰民事件</CALLOUTMESSAGE>
    <TASK>
        <TEXT>一名 ~o~流浪汉~s~ 正在制造骚乱.请与以下人员交谈:</TEXT>
        <TEXT>与呼叫求助的 ~g~保安~s~ 交谈.</TEXT>
        <TEXT>与呼叫求助的 ~g~行人~s~ 交谈.</TEXT>
        <TEXT>他们.</TEXT>
    </TASK>
    <SCENARIO>
        <SEARCHINGTRASH>
            <COOPERATIVE>
                <TEXT>~y~警员:~s~ 你好.一切都还好吗?你正在翻找这里的垃圾桶.</TEXT>
                <TEXT>~o~流浪汉:~s~ 是啊……只是找找瓶子,没别的.没惹麻烦.</TEXT>
                <TEXT>~y~警员:~s~ 我理解.但这里是私人领地.你能为我换个地方吗?</TEXT>
                <TEXT>~o~流浪汉:~s~ 哦……抱歉,我不知道.我没想惹任何人生气.</TEXT>
                <TEXT>~y~警员:~s~ 没关系.你需要什么吗?一杯咖啡?或是需要社会服务的帮助?</TEXT>
                <TEXT>~o~流浪汉:~s~ 你真好心,但是不用了……我现在就走.</TEXT>
                <TEXT>~y~警员:~s~ 好的.照顾好自己.</TEXT>
                <TEXT>~o~流浪汉:~s~ 谢谢.你也是.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~警员:~s~ 打扰一下.你不能待在这里,这里是私人领地.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我在这里待了好几天了.我没打扰任何人.</TEXT>
                <TEXT>~y~警员:~s~ 有投诉.我需要你离开.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我哪儿也不去.这是我唯一能待的地方.</TEXT>
                <TEXT>~y~警员:~s~ 如果保安来了,事情可能会对你变得不利.</TEXT>
                <TEXT>~o~流浪汉:~s~ 让他们来好了.我就待在这里.</TEXT>
                <TEXT>~y~警员:~s~ 我这是在给你一个公平的机会.</TEXT>
                <TEXT>~o~流浪汉:~s~ 而我说不.让我一个人待着.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~警员:~s~ 嘿,你能停一下吗?有人投诉你在这里翻找东西.</TEXT>
                <TEXT>~o~流浪汉:~s~ 投诉?投诉什么?我只是想活下去.别管我.</TEXT>
                <TEXT>~y~警员:~s~ 我希望和平解决.请为我换个地方.</TEXT>
                <TEXT>~o~流浪汉:~s~ 不.我哪儿也不去.</TEXT>
                <TEXT>~y~警员:~s~ 最后警告.别把事情搞复杂.</TEXT>
                <TEXT>~o~流浪汉:~s~ 复杂?你吓不到我.</TEXT>
                <TEXT>~y~警员:~s~ 退后.现在.</TEXT>
                <TEXT>~o~流浪汉:~s~ 够了!我要揍你!</TEXT>
                <TEXT>~y~警员:~s~ 住手!立刻!</TEXT>
            </FIGHT>
        </SEARCHINGTRASH>
        <BEGFORMONEY>
            <INTRO>
                <TEXT>~y~警员:~s~ 你好,我是为那个报警电话来的.你报告说有人骚扰你?</TEXT>
                <TEXT>~g~市民:~s~ 是的,警官.那边那个流浪汉一直在向我要钱.</TEXT>
                <TEXT>~y~警员:~s~ 能告诉我具体发生了什么吗?</TEXT>
                <TEXT>~g~市民:~s~ 我当时只是路过,忙我自己的事,他突然不知从哪儿冒出来拦住了我.</TEXT>
                <TEXT>~g~市民:~s~ 他一直坚持,说他‘比我更需要钱’.他根本不接受拒绝.</TEXT>
                <TEXT>~y~警员:~s~ 他威胁你或者以任何方式碰你了吗?</TEXT>
                <TEXT>~g~市民:~s~ 没有,并没有……但他太咄咄逼人了.让我很不舒服.</TEXT>
                <TEXT>~g~市民:~s~ 我知道日子不好过,但我不应该因为走在街上就被骚扰.</TEXT>
                <TEXT>~y~警员:~s~ 好的,我去和他谈谈.谢谢你告知情况.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~警员:~s~ 嘿,我需要和你谈谈.你刚才向那位女士要钱了?</TEXT>
                <TEXT>~o~流浪汉:~s~ 是的……我只问了一次.我不是有意吓她的.</TEXT>
                <TEXT>~y~警员:~s~ 她感到不舒服.你不能那样接近别人.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我懂,我懂……我没想惹麻烦.</TEXT>
                <TEXT>~y~警员:~s~ 很好.我需要你离开这个区域,别再向陌生人要钱了.</TEXT>
                <TEXT>~o~流浪汉:~s~ 好吧……我这就走.只是想混口饭吃,你懂的吧?</TEXT>
                <TEXT>~y~警员:~s~ 我理解.但你不能在这里这样做.</TEXT>
                <TEXT>~o~流浪汉:~s~ 好的……我走了.谢谢你没把事情闹大.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~警员:~s~ 先生,我需要和你谈谈.你一直在向别人要钱.</TEXT>
                <TEXT>~o~流浪汉:~s~ 那又怎样?我没伤害任何人.我只是问问.</TEXT>
                <TEXT>~y~警员:~s~ 你让一位女士感到不适.你需要离开.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我哪儿也不去.人们整天走来走去,我可以问任何我想问的人.</TEXT>
                <TEXT>~y~警员:~s~ 事情不是这样的.你不能骚扰路人.</TEXT>
                <TEXT>~o~流浪汉:~s~ 骚扰?我几乎没说什么.她反应过度了.</TEXT>
                <TEXT>~y~警员:~s~ 这是你最后的警告：停止在这里乞讨并离开这个区域.</TEXT>
                <TEXT>~o~流浪汉:~s~ 不.我留下.我没别的地方可去.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~警员:~s~ 我需要你停止向别人要钱来打扰他们.你不能在这里这样做.</TEXT>
                <TEXT>~o~流浪汉:~s~ 打扰?我只是问了个简单的问题.人们都太他妈敏感了.</TEXT>
                <TEXT>~y~警员:~s~ 她打电话给我们是因为她感到了威胁.你现在需要离开.</TEXT>
                <TEXT>~o~流浪汉:~s~ 离开?你怎么不自己走开?</TEXT>
                <TEXT>~y~警员:~s~ 别让事态升级.直接走开就行.</TEXT>
                <TEXT>~o~流浪汉:~s~ 走开?我受够了被推来推去!</TEXT>
                <TEXT>~y~警员:~s~ 嘿——冷静点.</TEXT>
                <TEXT>~o~流浪汉:~s~ 不!我受够听你的了!</TEXT>
                <TEXT>~o~流浪汉:~s~ 那就来吧!让我看看你有什么本事!</TEXT>
                <TEXT>~y~警员:~s~ 住手!别这样!</TEXT>
            </FIGHT>
        </BEGFORMONEY>
        <SLEEPONSTREET>
            <INTRO>
                <TEXT>~y~警员:~s~ 你好,我来回应你的报警电话.你说有人躺在地上?</TEXT>
                <TEXT>~g~市民:~s~ 是的,警官.就在那边.他已经在街上睡了至少一个小时了.</TEXT>
                <TEXT>~y~警员:~s~ 你看到他动过吗?有任何他可能受伤的迹象吗?</TEXT>
                <TEXT>~g~市民:~s~ 没有……我不确定他是否还好.所以我打了电话.看起来不太正常.</TEXT>
                <TEXT>~y~警员:~s~ 我理解.谢谢你告诉我们.我们会去查看他并确保他安全.</TEXT>
                <TEXT>~g~市民:~s~ 我只是不想他被车撞到什么的.这里的车离得很近.</TEXT>
                <TEXT>~y~警员:~s~ 你做得对.接下来交给我,看看他是需要医疗帮助还是只需要换个地方.</TEXT>
                <TEXT>~g~市民:~s~ 好的……谢谢你,警官.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~警员:~s~ 嘿,你能为我醒一下吗?我需要查看一下你的情况.</TEXT>
                <TEXT>~o~流浪汉:~s~ 呃……?什么……?哦,警官……我一定是睡着了.</TEXT>
                <TEXT>~y~警员:~s~ 有人打电话因为担心你的健康.你当时就躺在街边.</TEXT>
                <TEXT>~o~流浪汉:~s~ 是啊……我累了.没想到这有什么大不了的.</TEXT>
                <TEXT>~y~警员:~s~ 我理解,但这里很危险.车流很近,你可能会受伤.</TEXT>
                <TEXT>~o~流浪汉:~s~ 好吧……好吧……我起来.不是有意惹麻烦的.</TEXT>
                <TEXT>~y~警员:~s~ 很好.下次尽量找个更安全的地方.</TEXT>
                <TEXT>~o~流浪汉:~s~ 会的.谢谢你查看我的情况.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~警员:~s~ 先生,为我醒一下.我需要确认你还好.</TEXT>
                <TEXT>~o~流浪汉:~s~ 什……?我很好……别管我……只是想睡觉.</TEXT>
                <TEXT>~y~警员:~s~ 有人打电话因为他们以为你可能受伤了.你就在马路边上.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我没受伤.我选了这里.很安静.我留下.</TEXT>
                <TEXT>~y~警员:~s~ 这地方不安全.你不能睡在这里,车子离得太近了.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我在更糟的地方睡过.我不走.</TEXT>
                <TEXT>~y~警员:~s~ 我给你机会自己换个地方.</TEXT>
                <TEXT>~o~流浪汉:~s~ 而我说不.就让我睡吧.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~警员:~s~ 先生,醒醒.我需要查看你的情况并确保你安全.</TEXT>
                <TEXT>~o~流浪汉:~s~ 搞什么……?你为什么要烦我?</TEXT>
                <TEXT>~y~警员:~s~ 有人担心你的状况.你当时就躺在车流旁边.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我在睡觉!那不是犯罪!</TEXT>
                <TEXT>~y~警员:~s~ 这是为你的安全着想.我需要你起来,去个更安全的地方.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我哪儿也不去!别管我!</TEXT>
                <TEXT>~y~警员:~s~ 冷静.别把事态升级到不必要的程度.</TEXT>
                <TEXT>~o~流浪汉:~s~ 不!我受够了被赶来赶去!</TEXT>
                <TEXT>~o~流浪汉:~s~ 离我远点!</TEXT>
                <TEXT>~y~警员:~s~ 别这样!立刻住手!</TEXT>
            </FIGHT>
        </SLEEPONSTREET>
        <TRESPASSING>
            <INTRO>
                <TEXT>~g~保安:~s~ 警官,谢谢你来得这么快.</TEXT>
                <TEXT>~g~保安:~s~ 我们那边有个流浪汉在睡觉.</TEXT>
                <TEXT>~g~保安:~s~ 他在这里待了一阵子了,而这个区域是私人领地.他不能待在这里.</TEXT>
                <TEXT>~g~保安:~s~ 我试着叫醒他,让他离开,但他几乎没反应.</TEXT>
                <TEXT>~g~保安:~s~ 说实话,我不确定他的状况.他可能没事,但我不想冒险.</TEXT>
                <TEXT>~g~保安:~s~ 你能跟他谈谈并确保他离开这里吗?</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~警员:~s~ 嘿,你好,能为我醒一下吗?保安打电话给我们因为你是在私人领地上.</TEXT>
                <TEXT>~o~流浪汉:~s~ 嗯……什么?抱歉,我不是有意惹麻烦的.</TEXT>
                <TEXT>~y~警员:~s~ 你没有麻烦,但你不能待在这里.他们也担心你的状况.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我懂了……我只是太累了.我这就走.</TEXT>
                <TEXT>~y~警员:~s~ 好的.如果你需要帮助或找个更安全的地方去,我可以为你指点一些服务机构.</TEXT>
                <TEXT>~o~流浪汉:~s~ 感谢,警官.我现在就走.</TEXT>
                <TEXT>~y~警员:~s~ 好的.照顾好自己,好吗?</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~警员:~s~ 先生,为我醒一下.你是在私人领地上,不能待在这里.</TEXT>
                <TEXT>~o~流浪汉:~s~ 为什么……为什么我就不能休息?我没打扰任何人.</TEXT>
                <TEXT>~y~警员:~s~ 保安担心你的健康,也担心你非法侵入的事实.</TEXT>
                <TEXT>~o~流浪汉:~s~ 这只是个睡觉的地方.我没破坏任何东西.</TEXT>
                <TEXT>~y~警员:~s~ 我理解你累了,但你仍然需要离开.这个区域不对公众开放.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我不明白为什么我必须离开……这里很安静.</TEXT>
                <TEXT>~y~警员:~s~ 我最后一次要求你——我们让事情简单点.你需要起来并离开这个区域.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~警员:~s~ 先生,醒醒.你非法侵入,不能待在这里.</TEXT>
                <TEXT>~o~流浪汉:~s~ 走开……我哪儿也不去!</TEXT>
                <TEXT>~y~警员:~s~ 保安担心出了什么事.我试图帮你,但你需要离开.</TEXT>
                <TEXT>~o~流浪汉:~s~ 帮?你们警察从来不帮人!就让我一个人待着!</TEXT>
                <TEXT>~y~警员:~s~ 冷静.我给你一个合法的命令,让你起来并离开这个区域.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我说了不!离我远点!</TEXT>
                <TEXT>~y~警员:~s~ 别做你会后悔的事——停止反抗!</TEXT>
            </FIGHT>
        </TRESPASSING>
        <FIREPLACE>
            <INTRO>
                <TEXT>~g~保安:~s~ 警官,谢谢你来.我们这里又有个流浪汉.</TEXT>
                <TEXT>~g~保安:~s~ 这次他居然点了一小堆火来取暖,就紧挨着建筑.</TEXT>
                <TEXT>~y~警员:~s~ 好的,那绝对不安全.周围还有其他人吗,还是只有他?</TEXT>
                <TEXT>~g~保安:~s~ 只有他,但烟雾触发了我们的一个传感器.而且我要求时他拒绝离开.</TEXT>
                <TEXT>~y~警员:~s~ 明白了.我会和他谈谈,控制住局面.</TEXT>
                <TEXT>~g~保安:~s~ 请务必.他不能待在这里,而且那火是个真正的隐患.</TEXT>
                <TEXT>~y~警员:~s~ 我理解.我会确保他把火扑灭并离开.</TEXT>
            </INTRO>
            <COOPERATIVE>
                <TEXT>~y~警员:~s~ 嘿,先生.我需要你为我从火边退开.</TEXT>
                <TEXT>~o~流浪汉:~s~ 哦……警官.我只是想取暖.</TEXT>
                <TEXT>~y~警员:~s~ 我理解,但这里是私人领地,而且那火是个严重的隐患.</TEXT>
                <TEXT>~o~流浪汉:~s~ 是啊……我很抱歉.不是有意惹任何麻烦.</TEXT>
                <TEXT>~y~警员:~s~ 没关系.我只需要你把火扑灭并离开.</TEXT>
                <TEXT>~o~流浪汉:~s~ 好吧,好吧.我走.我不想惹任何麻烦.</TEXT>
                <TEXT>~y~警员:~s~ 很好.照顾好自己,在外面注意安全.</TEXT>
            </COOPERATIVE>
            <NONCOMPLIANT>
                <TEXT>~y~警员:~s~ 先生,我需要你把火扑灭并离开这个区域.</TEXT>
                <TEXT>~o~流浪汉:~s~ 为什么?我没伤害任何人.我只是需要温暖.</TEXT>
                <TEXT>~y~警员:~s~ 这是私人领地,你不能待在这里.这火也很危险.</TEXT>
                <TEXT>~o~流浪汉:~s~ 火势控制住了.我知道我在做什么.</TEXT>
                <TEXT>~y~警员:~s~ 那并没有改变情况.你需要离开.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我不明白为什么我必须离开.就一晚上而已.</TEXT>
                <TEXT>~y~警员:~s~ 我给你一个合法的命令.别把事情搞得比它必须的更复杂.</TEXT>
            </NONCOMPLIANT>
            <FIGHT>
                <TEXT>~y~警员:~s~ 先生,警察.从火边退开,和我谈谈.</TEXT>
                <TEXT>~o~流浪汉:~s~ 滚开……我哪儿也不去!</TEXT>
                <TEXT>~y~警员:~s~ 这是私人领地,那火是个隐患.你现在需要把它扑灭.</TEXT>
                <TEXT>~o~流浪汉:~s~ 我不听你的命令!离我远点!</TEXT>
                <TEXT>~y~警员:~s~ 冷静.我试图和平解决这件事.</TEXT>
                <TEXT>~o~流浪汉:~s~ 和平?你来就是为了把我赶走!算了吧!</TEXT>
                <TEXT>~y~警员:~s~ 别这样.把你的手放在我能看见的地方——立刻!</TEXT>
            </FIGHT>
        </FIREPLACE>
    </SCENARIO>
</CALLOUT>